src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/EventType.java

```diff
@@ -231,6 +233,21 @@
         this.notifiable = notifiable;
     }
 
+    public static EventType getEventTypeByCcdType(String ccdType) {
+        EventType e = null;
+        for (EventType event : EventType.values()) {
+            if (event.getCcdType().equals(ccdType)) {
+                e = event;
+            }
+        }
+        return e;
+    }
+
+    @JsonCreator
```

Comment:
why do we have 2 methods for the same purpose?

Reply:
there's only one. getEventTypeByCcdType